src/pretix/presale/templates/pretixpresale/event/fragment_product_list.html

```diff
@@ -6,12 +6,12 @@
 {% load eventsignal %}
 {% load rich_text %}
 {% for tup in items_by_category %}
-    <section {% if tup.0 %}aria-labelledby="category-{{ tup.0.id }}"{% else %}aria-label="{% trans "Uncategorized items" %}"{% endif %}{% if tup.0.description %} aria-describedby="category-info-{{ tup.0.id }}"{% endif %}>
-        {% if tup.0 %}
-            <h3 id="category-{{ tup.0.id }}">{{ tup.0.name }}</h3>
-            {% if tup.0.description %}
-                <div id="category-info-{{ tup.0.id }}">{{ tup.0.description|localize|rich_text }}</div>
-            {% endif %}
+    <section aria-labelledby="category-{{ tup.0.id|default:0 }}"{% if tup.0.description %} aria-describedby="category-info-{{ tup.0.id }}"{% endif %}>
```

Comment:
We are now using |default:0 sometimes and sometimes not for the same field. Either we need it or we don't ;)

Reply:
nevermind